ui/src/routes/_layout/create-organization.tsx

```diff
@@ -104,7 +104,7 @@ const CreateOrganizationPage = () => {
                     <Input {...field} autoFocus />
                   </FormControl>
                   <FormDescription>
-                    Enter the name of your organization
+                    The name of the organization.
```

Comment:
While adjusting the descriptions, it would make sense to remove redundant ones. If the field in org creation form has a visible title of `name`, description like this is totally redundant.

IMO [this](https://primer.style/ui-patterns/forms/overview#caption) is quite a reasonable guideline.

Reply:
Done in a new commit as it was easier to filter these out after the "normalization" of descriptions.